src/AppInstallerCommonCore/Public/AppInstallerVersions.h

```diff
@@ -75,9 +91,48 @@ namespace AppInstaller::Utility
         // Gets the part breakdown for a given version; used for tests.
         const std::vector<Part>& GetParts() const { return m_parts; }
 
+        // Returns if the version is an approximate version.
+        bool IsApproximateVersion() const { return m_approximateComparator != ApproximateComparator::None; }

```

Comment:
```suggestion
        bool IsApproximate() const { return m_approximateComparator != ApproximateComparator::None; }
```
Why use many word, when few word do trick?